extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -194,6 +200,7 @@ private static void doServerStart(Vertx vertx, HttpConfiguration httpConfigurati
             Supplier<Integer> eventLoops)
             throws IOException {
         // Http server configuration
+        router.route().failureHandler(new QuarkusErrorHandler(launchMode.isDevOrTest()));
```

Comment:
It might be better to register it as "last" explicitly.

Reply:
Won't this be last by default as the router has already been setup?

Reply:
I have updated it anyway